lnwallet/channel.go

```diff
@@ -2992,7 +2989,8 @@ func (lc *LightningChannel) SignNextCommitment() (lnwire.Sig, []lnwire.Sig, erro
 //  * RevokeAndAck: if we sent a revocation message that they claim to have
 //    not received
 func (lc *LightningChannel) ProcessChanSyncMsg(
-	msg *lnwire.ChannelReestablish) ([]lnwire.Message, error) {
+	msg *lnwire.ChannelReestablish) ([]lnwire.Message, []channeldb.CircuitKey,
```

Comment:
The `godoc` comment above was never documented to describe the new additional parmaters.